nelmio/api-doc-bundle/3.0/config/routes/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,3 @@
+NelmioApiDocBundle:
+    resource: "@NelmioApiDocBundle/Resources/config/routing/swaggerui.xml"
+    prefix:   /api/doc
```

Comment:
I guess this should be commented and we should mention that it requires the asset component ans the twig bundle.